app/views/provider/admin/dashboards/_services_navigation.html.slim

```diff
@@ -1,24 +1,9 @@
 nav.DashboardNavigation.DashboardNavigation--services
   ul.DashboardNavigation-list
-    /- if can? :manage, :partners
-
+    - if can?(:manage, :service_contracts) && !current_account.master?
```

Comment:
Imo it is better `- if can?(:manage, :service_contracts) && current_account.tenant?`

Reply:
Independently on what's better, it exists:
https://github.com/3scale/porta/blob/74a67bf651b7843e1dcd3367aa9e6c091f4d40ec/app/models/account.rb#L383-L385

Imo it is better because we want to show it only for tenants and it is more easily readable for me. But both are fine anyway 😄

Reply:
damn Atom: 
<img width="973" alt="screenshot 2018-11-05 10 46 18" src="https://user-images.githubusercontent.com/54224/47990563-1ee29380-e0e8-11e8-907f-51f8bc8d7910.png">